cirq-core/cirq/ops/classically_controlled_operation_test.py

```diff
@@ -498,9 +506,9 @@ def test_scope_local():
     cirq.testing.assert_has_diagram(
         cirq.Circuit(outer_subcircuit),
         """
-      [       [ 0: ───M───X─── ]             ]
-0: ───[ 0: ───[       ║   ║    ]──────────── ]────────────
-      [       [ a: ═══@═══^═══ ](loops=2)    ](loops=2)
+      [       [ 0: ───M───X─── ]                               ]
+0: ───[ 0: ───[       ║   ║    ]────────────────────────────── ]──────────────────────────────
+      [       [ a: ═══@═══^═══ ](repetition_ids=['0', '1'])    ](repetition_ids=['0', '1'])
```

Comment:
This might get quite long for large repetition count.
Should we perhaps use `(loops=2, use_repetition_ids=True)` instead?

PS: No strong preference on my side.

Reply:
Agreed. Changed to use `(loops=2, use_repetition_ids=True)` as suggested.